core-plugins/src/e2e-test/features/joiner/Joiner.feature

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Joiner
+Feature: Joiner analytics - Verify File source data transfer using Joiner analytics
+  @HDF_FILE_SOURCE_TEST @HDF_FILE_SOURCE_JOINER_TEST @HDF_FILE_SINK_TEST @HDF_FILE_SINK_JOINER_TEST
```

Comment:
Why do we have `HDF` prefix in these tags? Are they specific to `HDF`?

Reply:
Since only `HDF` ones need to have a GCS bucket otherwise we could just use local file for testing.

Reply:
Got it, thanks for the info.

May be we should then create a new `Test Runner` which can run tests specific to HDF because once we start adding more tests using local file they will start failing in `HDF`.

`Test Runner` can be made configurable through `pom.xml`.
See this PR for ref - https://github.com/data-integrations/google-cloud/pull/1067

Reply:
But for local file, they can use `@FILE_SOURCE or @FILE_SINK` as the tag in scenario.

Reply:
After a second thought, I think file on GCS could also be used for sandbox testing. I have changed the tag name to `GCS_xxx` in the end.